package.json

```diff
@@ -1,19 +1,19 @@
 {
   "name": "iter-tools",
-  "version": "3.0.0",
+  "version": "4.0.0",
   "description": "The iterator toolbox",
-  "main": "index.js",
-  "typings": "index.d.ts",
+  "main": "lib/index.js",
+  "module": "lib/index.mjs",
+  "typings": "lib/index.d.ts",
   "scripts": {
     "test": "npm run build && mocha tests/**/*.js",
     "benchmarks": "node benchmarks",
     "watch": "npm run test -- -w",
-    "lint": "eslint --fix --ext .js ./lib ./tests ./benchmarks",
-    "build": "babel --plugins transform-runtime lib -d es5",
-    "release:major": "npm-release major",
-    "release:minor": "npm-release minor",
-    "release:patch": "npm-release patch",
-    "precommit": "npm run build && npm run lint",
+    "lint": "eslint --fix --ext .mjs ./src ./tests ./benchmarks",
+    "clean": "rm -rf lib/*.* && rm -rf lib/async/*.* && rm -rf es5/*.* && rm -rf es5/async/*.* && rm -rf package/*",
```

Comment:
I don't destroy my directory structure, so that I don't need to have mkdirp's everywhere. Then again, mkdirp isn't all that annoying.